src/main/java/com/uid2/shared/auth/OperatorKey.java

```diff
@@ -26,14 +27,15 @@ public OperatorKey(String key, String name, String contact, String protocol, lon
         this.siteId = null;
```

Comment:
might want to hardcode Role.OPERATOR here for backward compatibility - the shared lib might be pushed to prod first while using old Core/Admin service versions so need to consider admin service might not be calling the new constructor below but this one.

Reply:
okay, got it